app/src/main/res/layout/activity_weather.xml

```diff
@@ -34,4 +34,15 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintTop_toBottomOf="@id/button_weather_activity_id_forecast" />
 
+    <ProgressBar
+        android:id="@+id/progressBar_weather_activity_data_load"
+        style="?android:attr/progressBarStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:visibility="invisible"
```

Comment:
please set it as gone, if you put it as invisible it is on memory but nos visible, if you put it as gone it is not in memory

Reply:
done